backend-spring-boot/src/main/java/com/pharmacy/cpis/util/validators/StringValidator.java

```diff
@@ -1,14 +1,18 @@
 package com.pharmacy.cpis.util.validators;
 
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
 public class StringValidator {
 
-    static public Boolean isEmail(String email) {
-        // TODO: find Java regex email
-        return true;
+    public static final String VALID_EMAIL_ADDRESS_REGEX = "^(.+)@(\\S+)$";
```

Comment:
Does this work if the email contains a "."? Since I think .+ in regex is every character, except for "." itself.

Reply:
I did not test that case, I just take some regex from StackOverflow and try some of the use cases (but not that one).